app/views/home/index.html.erb

```diff
@@ -1,37 +1,13 @@
+<!--<meta http-equiv="refresh" content="2" />-->
 <div class="jumbotron">
-  <h1 class="display-4">Connections Portal</h1>
+      <h1 class="display-4">Connections Portal</h1>
   <p class="lead">A digital tool for organizing and arranging networks, relationships and human connections.</p>
   <hr>
   <p>We hope you enjoy using it!
-    <a href="https://github.com/hpi-swt2/connections-portal/issues/new" target="_blank">Give feedback!</a></p>
+        <a href="https://github.com/hpi-swt2/connections-portal/issues/new" target="_blank">Give feedback!</a></p>
   <p class="lead" id="registration">
     <%# fa_icon helper: https://github.com/bokmann/font-awesome-rails %>
```

Comment:
Comment? Remove

Reply:
+1

Reply:
We haven't added this one, maybe it's meant as a reference for the use of the icons.